assets/js/components/consent-mode/ConfirmDisableConsentModeDialog.js

```diff
@@ -97,6 +97,7 @@ export default function ConfirmDisableConsentModeDialog( {
 
 	return (
 		<ModalDialog
+			className="googlesitekit-settings-module__confirm-disable-consent-mode-modal"
```

Comment:
Please correct me if I'm wrong but I don't see this class name defined in any stylesheet. I think this should be changed to `googlesitekit-settings-module__confirm-disconnect-modal` instead.

```suggestion
			className="googlesitekit-settings-module__confirm-disconnect-modal"
```

Reply:
Sorry, it was my mistake @nfmohit . 
The correct class name is the one you suggested. I have added that change.